draft-ietf-rtcweb-jsep.xml

```diff
@@ -1977,13 +1977,224 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
+                is stored.</t>
+              <t>TODO: msid-semantic, identity, extmap and related values</t>
+            </list>
+            Once all the session-level lines have been parsed, processing
+            continues with the lines in media sections.
+          </t>
+        </section>
+        <section title="Media Section Parsing">
+          <t>Like the session-level lines, the media session lines MUST occur
+            in the specific order and with the specific syntax defined in
+            <xref target="RFC4566"/>, Section 5.</t>
+          <t>Parse m=:
+            <list style="symbols">
+              <t>Check port (i.e. rejected), read protocol</t>
+              <t>Read codec ordering</t>
+            </list>
+          </t>
+          <t>Ignore c=</t>
+          <t>Parse b= as max send bandwidth</t>
+          <t>Parse the following a=:
+            <list style="symbols">
+              <t>Read mid as id for desc</t>
+              <t>Read ICE ufrag/pwd and save</t>
+              <t>Read ICE options</t>
+              <t>Read fingerprint and setup</t>
+              <t>Handle RTP attributes:
+                <list style="symbols">
+                  <t>Read fmtp attributes</t>
+                  <t>Read ssrc-group attributes</t>
+                  <t>Read ssrc attributes</t>
+                  <t>Read ptime, maxptime</t>
+                  <t>Read header extensions</t>
+                  <t>Read rtcp-fb</t>
+                  <t>Read direction attribs (recvonly, etc)</t>
+                  <t>Read rtcp-mux</t>
+                </list>
+              </t>
+              <t>Handle SCTP attributes:
+                <list style="symbols">
+                  <t>Read sctpmap/port attributes</t>
+                </list>
+              </t>
+            </list>
+          </t>
+        </section>
+        <section title="Semantics Verification">
+          <t>Assuming parsing completes successfully, the parsed description is
+            then evaluated to ensure internal consistency as well as proper support
+            for mandatory features. Specifically, the following checks are
+            performed:
+              <list style="symbols">
+                <t>For each m= section, valid values for each of the
+                mandatory-to-use features enumerated in <xref target="sec.usage-requirements"/> MUST be present. These values MAY either be present at the
+                media level, or inherited from the session level.
+                  <list style="symbols">
+                    <t>ICE ufrag and password values</t>
+                    <t>DTLS fingerprint and setup values</t>
+                  </list>
+                </t>
+                <t>If a BUNDLE group is present, the referenced m= sections MUST
+                exist. If multiple BUNDLE groups exist, each referenced m=
+                section MUST only exist in a single BUNDLE group. [TODO: should this apply to all groups?]</t>
```

Comment:
I couldn't find a good reference in BUNDLE but it is covered in 5888, I can just refer to that section instead.